src/Elevator.java

```diff
@@ -204,25 +246,37 @@ public void run() {
 	 * @throws InterruptedException
 	 */
 	private void move() throws InterruptedException {
-
+		int startFloor = this.getCurrentFloor(); // Saves the position of the elevator before attempting a move.
 		this.updateCurrentFloor();
-		System.out.println("Elevator "+elID+": At floor: " + getCurrentFloor());
-		Thread.sleep(2000);
+		Thread.sleep(3000);
+		if (this.getCurrentFloor() == startFloor) { // If the elevator has not moved after 3 seconds, it is stuck and
+													// therefore is hard faulting
+			System.out.println("Elevator " + this.getElID()+ "in hard fault state");
+//			currState = HARDFAULTSTATE;
```

Comment:
This line will need to be changed and uncommented when the hard fault state is implemented